app/src/main/java/com/example/campusguide/directions/Route.kt

```diff
@@ -19,18 +23,58 @@ import kotlinx.coroutines.launch
  * @param end The name of the location where the route ends
  */
 class Route constructor(private val map: GoogleMap, private val activity: AppCompatActivity) {
+    private val PATTERN_DASH_LENGTH_PX = 20
+    private val PATTERN_GAP_LENGTH_PX = 20
+    private val DASH: PatternItem = Dash(PATTERN_DASH_LENGTH_PX.toFloat())
+    private val GAP: PatternItem = Gap(PATTERN_GAP_LENGTH_PX.toFloat())
+    private val PATTERN_POLYGON_ALPHA = listOf(GAP, DASH)
+    private val COLOR_BLUE_ARGB = 0xff0000ff
     private var polyline: Polyline? = null
+    private var begin: Marker? = null
+    private var dest: Marker? = null
+
+    private fun capitalizeWords(location: String): String {
```

Comment:
I feel like this function should be in a utility class, not in the Route class.

Maybe make a new StringUtils class for it to reside in?